editor/plugins/style_box_editor_plugin.cpp

```diff
@@ -53,35 +60,86 @@ void StyleBoxPreview::edit(const Ref<StyleBox> &p_stylebox) {
 		preview->add_theme_style_override("panel", stylebox);
 		stylebox->connect("changed", callable_mp(this, &StyleBoxPreview::_sb_changed));
 	}
+	Ref<StyleBoxTexture> sbt = p_stylebox;
+	grid_preview->set_visible(sbt.is_valid());
 	_sb_changed();
 }
 
 void StyleBoxPreview::_sb_changed() {
 	preview->update();
 }
 
+void StyleBoxPreview::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE:
+		case NOTIFICATION_THEME_CHANGED: {
+			if (!is_inside_tree()) {
+				// TODO: This is a workaround because `NOTIFICATION_THEME_CHANGED`
+				// is getting called for some reason when the `TexturePreview` is
+				// getting destroyed, which causes `get_theme_font()` to return `nullptr`.
+				// See https://github.com/godotengine/godot/issues/50743.
+				break;
+			}
```

Comment:
Weird, I never seen this issue and never seen such code in any editor.

Reply:
See https://github.com/godotengine/godot/pull/50745